docs/policy/topology-aware.md

```diff
@@ -369,3 +369,22 @@ the amount of memory for containers in `Burstable` pods or run the
 [resource-annotating webhook](../webhook.md) to provide `cri-resmgr` with
 an exact copy of the resource requirements from the Pod Spec as an extra
 Pod annotation.
+
+## Reserved pool namespaces
+
+User is able to mark certain namespaces to have reserved CPU allocation.
```

Comment:
I think we should explain a bit more (a few sentences max.) what this means in practice. Maybe something like "Containers belonging to such namespaces will only run on CPUs set aside according to the global CPU reservation, as configured by the `ReservedResources` configuration option in the `policy` section.